torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -649,55 +740,88 @@ void PyTorchModelLoader::loadConstant(const torch::jit::Node *ptNode) {
     const auto &valsList = iVal.toBoolList();
     const auto &vals = c10::impl::toVector(valsList);
     if (vals.empty()) {
-      return;
+      llvm::Error::success();
     }
     t.reset(glow::ElemKind::BoolTy, {vals.size()});
     auto handle = t.getHandle<bool>();
     for (size_t i = 0; i < vals.size(); ++i) {
       handle.raw(i) = vals[i];
     }
   } else if (iVal.isTensor()) {
-    assert(false && "not yet implemented tensor constants");
+    RETURN_ERR("Not yet implemented tensor constants.");
   } else if (iVal.isNone()) {
     // Nothing to do for None
-    return;
+    llvm::Error::success();
   } else {
-    assert(false && "Unsupported constant type");
+    RETURN_ERR("Unsupported constant type.");
   }
 
   glow::Constant *glowConstant = mod->createConstant("constant", std::move(t));
-  addGlowNodeValue(outputs[0], glowConstant->getOutput());
+  return addGlowNodeValue(outputs[0], glowConstant->getOutput());
+}
+
+/*static*/
+llvm::Error PyTorchModelLoader::loadJITGraph(
+    glow::Function &F, torch::jit::Graph &subgraph,
+    at::ArrayRef<torch::jit::IValue> &inputs,
+    std::vector<glow::Placeholder *> &inputPlaceholders,
+    std::vector<glow::Placeholder *> &outputPlaceholders) {
+  llvm::Error error = llvm::Error::success();
```

Comment:
`MARK_ERR_CHECKED`

Reply:
Can you explain how to use MARK_ERR_CHECKED?

Reply:
just add `MARK_ERR_CHECKED(error)` below `llvm::Error error = llvm::Error::success();`
`llvm::Error::success()` creates a new Error and assigns it to `error` but then when `error` is passed by ref into the PyTorchModelLoader constructor, it will be overwritten and the original Error created by `llvm::Error::success()` will be destroyed and all llvm::Errors have to be checked before they're destroyed so you can just check it up front (kinda a hack) using `MARK_ERR_CHECKED`